libraries/door43_tools/project_deployer.py

```diff
@@ -126,7 +126,7 @@ def deploy_revision_to_door43(self, build_log_key):
                     content += '<h1 class="conversion-requested">{0}</h1>'.format(build_log['message'])
                     content += '<p><i>No content is available to show for {0} yet.</i></p>'.format(repo_name)
                     content += """
-                    <script type="text/javascript">setTimeout(function(){window.location.reload(1);}, 10000);</script>
+                    <script type="text/javascript">CONV_REQUESTED=setTimeout(function(){window.location.reload(1);}, 10000);</script>
```

Comment:
Misunderstood what you meant.  I was trying to delete it at runtime which didn't work since the timer was already started before the script could be deleted.  Then thought the only way to cancel the timer after starting is if we have the timer ID (saved in CONV_REQUESTED).  So then I check for the CONV_REQUESTED when the page loads and cancel it with javascript and do periodic checking of the build_log.json.  Realize now I was making the whole thing too complicated.